modules/analysis-runner/src/cljdoc/analysis/deps.clj

```diff
@@ -103,15 +102,18 @@
 
 (defn resolved-and-cp
   "Build a classpath for the project specified by `pom-url`."
-  [pom-url repos]
+  [jar-url pom-url repos]
   {:pre [(string? pom-url)]}
   (println pom-url repos)
   (let [pom (pom/parse (slurp pom-url))
+        {:keys [group-id artifact-id version]} (pom/artifact-info pom)
+        project (symbol group-id artifact-id)
```

Comment:
You could use `cljdoc.util/clojars-id` here but it doesn't really matter much.

Reply:
```clj
(util/clojars-id (pom/artifact-info pom))
```
Would give you a symbol that you can use in the deps map.

Reply:
Thanks for the tip!